packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -22,6 +22,76 @@ import {
   ensureWebhook,
 } from './ensureTarget';
 
+export type Alert = NonNullable<
+  NonNullable<Types.GetAlertsQuery['alert']> extends Array<infer T> ? T : never
+>;
+
+export type ConnectedWallet = NonNullable<
+  NonNullable<Types.GetConnectedWalletsQuery['connectedWallet']> extends Array<
+    infer T
+  >
+    ? T
+    : never
+>;
+
+export type EmailTarget = NonNullable<
+  NonNullable<Types.GetEmailTargetsQuery['emailTarget']> extends Array<infer T>
+    ? T
+    : never
+>;
+
+export type Filter = NonNullable<
+  NonNullable<Types.GetFiltersQuery['filter']> extends Array<infer T>
+    ? T
+    : never
+>;
+
+export type SmsTarget = NonNullable<
+  NonNullable<Types.GetSmsTargetsQuery['smsTarget']> extends Array<infer T>
+    ? T
+    : never
+>;
+
+export type Source = NonNullable<
+  NonNullable<Types.GetSourcesQuery['source']> extends Array<infer T>
+    ? T
+    : never
+>;
+
+export type TargetGroup = NonNullable<
+  NonNullable<Types.GetTargetGroupsQuery['targetGroup']> extends Array<infer T>
+    ? T
+    : never
+>;
+
+export type TelegramTarget = NonNullable<
+  NonNullable<Types.GetTelegramTargetsQuery['telegramTarget']> extends Array<
+    infer T
+  >
+    ? T
+    : never
+>;
+
+export type DiscordTarget = NonNullable<
+  NonNullable<Types.GetDiscordTargetsQuery['discordTarget']> extends Array<
+    infer T
+  >
+    ? T
+    : never
+>;
+
+export type FrontendClientData = {
+  alerts: ReadonlyArray<Alert>;
+  connectedWallets: ReadonlyArray<ConnectedWallet>;
+  emailTargets: ReadonlyArray<EmailTarget>;
+  filters: ReadonlyArray<Filter>;
+  smsTargets: ReadonlyArray<SmsTarget>;
+  sources: ReadonlyArray<Source>;
+  targetGroups: ReadonlyArray<TargetGroup>;
+  telegramTargets: ReadonlyArray<TelegramTarget>;
+  discordTargets: ReadonlyArray<DiscordTarget>;
+};
```

Comment:
if we add a separate `fetchData` query, then we can simply expose the generated GraphQL type directly rather than having to re-declare the types here.

Reply:
Addressed! Export the `FetchDataQuery` as `FrontendClientData`